cmake/modules/LICENSE-ocaml-cmake-0BSD.txt

```diff
@@ -0,0 +1,12 @@
+Copyright (C) 2018 whitequark@whitequark.org
```

Comment:
Is this MIT or BSD as the file name says? I am not keen on including other licenses even if they are fine. 
It adds to the pile of questions users may feel compelled to consider for due diligence when integrating 
external software, such as Z3.